src/Common/tests/System/Collections/ICollection.NonGeneric.Tests.cs

```diff
@@ -208,7 +208,7 @@ public void ICollection_NonGeneric_CopyTo_TwoDimensionArray_ThrowsException(int
             }
         }
 
-        [ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNonZeroLowerBoundArraySupported))]
+        [ConditionalTheory("System.PlatformDetection, CoreFx.Private.TestUtilities!" + nameof(PlatformDetection.IsNonZeroLowerBoundArraySupported))]
```

Comment:
I would prefer we not merge this PR to avoid the unnecessary churn. 

you cannot use MethodInfo/PropertyInfo directly in a custom attribute. Custom attributes only support primitive types and "System.Type" arguments.